hw/ip/dcd/data/dcd.hjson

```diff
@@ -153,10 +159,10 @@
 	    name: "cond",
 	    desc: "1-bit for the condition; 1'b0 means min<=ADC<=max, 1'b1 means ADC>max or ADC<min ",
 	  }
-          { bits: "17:16",
-	    name: "max_ext",
-	    desc: "2-bit extension; RO 0",
-	  }
+//          { bits: "17:16",
```

Comment:
@esshiu i'll file a separate issue for you to restore these and connect the unused signals.